docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -118,188 +128,212 @@ export default function PlainCssButtonDeep() {
 Explicitly providing the class names to the component is too much effort?
 [You can target the class names generated by Material-UI](/styles/advanced/#with-material-ui-core).
 
-[![Edit Button](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/global-css-bir9e)
+[![Edit Button](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/global-classnames-mc4wl)
 
-**GlobalCssButton.css**
+**GlobalCssSlider.css**
 
 ```css
-.MuiButton-root {
-  background-color: #6772e5;
-  box-shadow: 0 4px 6px rgba(50, 50, 93, 0.11), 0 1px 3px rgba(0, 0, 0, 0.08);
-  padding: 7px 14px;
-}
-.MuiButton-root:hover {
-  background-color: #5469d4;
+.MuiSlider-root {
+  color: #20b2aa;
 }
-.MuiButton-label {
-  color: #fff;
+.MuiSlider-root:hover {
+  color: #2e8b57;
 }
 ```
 
-**GlobalCssButton.js**
+**GlobalCssSlider.js**
 
 ```jsx
 import * as React from 'react';
-import Button from '@material-ui/core/Button';
-import './GlobalCssButton.css';
+import Slider from '@material-ui/lab/SliderStyled';
+import './GlobalCssSlider.css';
 
-export default function GlobalCssButton() {
-  return <Button>Customized</Button>;
+export default function GlobalCssSlider() {
+  return <Slider defaultValue={30} />;
 }
 ```
 
 ### Controlling priority ⚠️
 
-**Note:** JSS injects its styles at the bottom of the `<head>`. If you don't want to mark style attributes with **!important**, you need to change [the CSS injection order](/styles/advanced/#css-injection-order), as in the demo:
+**Note:** Most CSS-in-JS solutions inject their styles at the bottom of the html `<head>`. If you don't want to mark style attributes with **!important**, you need to change the CSS injection order. Here is a demo of how it can be done for our default styled engine - emotion:
```

Comment:
```suggestion
**Note:** Most CSS-in-JS solutions inject their styles at the bottom of the HTML `<head>`. If you don't want to mark style attributes with **!important**, you need to change the CSS injection order. Here's a demo of how it can be done for the default styled engine - emotion:
```